src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,100 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
 import { formatNumber, Metric } from '../lib';
+import { MetricMeta } from '../lib/load_status';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: any;
+  description: any;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
```

Comment:
I ended up using a combination of the suggestions from design here, using `EuiStat` as a footer to the existing `EuiCard` in `MetricTile`. `EuiCard` with horizontal layout doesn't support a footer so I had to go with aligning the text instead.